Library/Homebrew/extend/pathname.rb

```diff
@@ -131,6 +131,12 @@ def write(content, *open_args)
     open("w", *open_args) { |f| f.write(content) }
   end
 
+  # Only appends to a file that is already created.
+  def write_append(content, *open_args)
+    raise "Cannot append file that doesn't exist: #{self}" unless exist?
+    open("a", *open_args) { |f| f.puts(content) }
```

Comment:
What is the justification for requiring file existence? Not saying it's a bad idea, but wondering what the rationale is for not making it equivalent to a shell `>>` append redirection?